lib/v2/ncu/jwc.js

```diff
@@ -0,0 +1,32 @@
+const got = require('@/utils/got'); // 自订的 got
+const cheerio = require('cheerio'); // 可以使用类似 jQuery 的 API HTML 解析器
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const baseUrl = "https://jwc.ncu.edu.cn";
+    const response = await got(baseUrl, { responseType: "text" });
```

Comment:
No need to specify default value.
https://github.com/sindresorhus/got/tree/v11.8.6#responsetype